### DISCUSSION TITLE
Deleting site data for some sites and not others

### DISCUSSION THREAD

**Discussion by buckeyered** [Cookie-AutoDelete/Cookie-AutoDelete#1592] [Q&A / Support]
I admit that I don't understand all the settings in cookie-autodelete and don't want to mess it up by randomly clicking on settings.
I have cookie-autodelete set to auto delete cookies that are not whitelisted but the site data remains behind, how can I get the site data to auto delete with the cookies? thanks sorry I just don't understand all the settings.

**Comment by kennethtran93** [comment]
Are you on Chrome or Firefox?  Sites are only cleaned when there aren't any more tabs open with that domain or on browser restart.

There's settings under the 'Other Browsing Data Cleanup Options' that you can check to enable its cleanup.

We currently only clean a subset of available browsing data types, due to API limitations and/or the time to add/implement the ones made available later on.
